src/faultlog/deconfig_records.cpp

```diff
@@ -85,17 +85,27 @@ int DeconfigRecords::getCount(const GuardRecords& guardRecords)
     DeconfigDataList deconfigList;
     pdbg_target_traverse(nullptr, getDeconfigTargets, &deconfigList);
     DeconfigDataList onlyDeconfigList;
+    // consider only those targets that are not part of guard list
```

Comment:
This code and the code in the populate are duplicate right? can we optimize it to avoid code duplication?

Reply:
done fixed

Reply:
I am still seeing half of the DeconfigRecords::populate and DeconfigRecords::getCount same